CMake/PCHSupport.cmake

```diff
@@ -12,6 +12,10 @@ option( PROJECTS_USE_PRECOMPILED_HEADERS "Use precompiled headers" TRUE )
 #--------------------------------------------------------------------------------------------------
 set( CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS true )
 
+if ( UNIX AND NOT APPLE )
```

Comment:
Move this at the option's initialisation

Reply:
Do you mean that I put :
```cmake
if ( UNIX AND NOT APPLE)
   option(.... FALSE)
else()
   option(... TRUE)
endif()
```
Because the way I used was the only one I knew of. Or maybe I should put that in another file ?

Reply:
Yes, at the top of the file.

Reply:
Ok thanks. Easy enough to fix.